### ISSUE TITLE
Allegro One Box/Punkt, UPS/DPD

### ISSUE THREAD

**Issue by ZAK28ZAK** [allegro/allegro-api#8995]
Dzień dobry,
Czy zmieniło się coś pod względem generowania biletów Allegro One Box/Punkt ?
Czy dotychczasowe metody wysyłek zostały zastąpione przez jakieś inne typu:
Allegro One Box, UPS
Allegro One Punkt, UPS
Allegro One Box, DPD
Allegro One Punkt, DPD

Pytam, bo na dotychczasowej metodzie przestały nam się generować bilety przewozowe, a czytałem, że od 17 kwietnia Allegro One jest realizowane i za pomocą UPS i za pomocą DPD.

Także czy stara metoda pod względem API została i tylko jej mimo wszystko trzeba używać, czy powstały jakieś nowe wysyłki dla tych nowych metod?

Jeśli nowe, to czy mogą Państwo podać ich nazwy wraz z powiązaniem API tych usług?

**Comment by MaciejFrackowiak** [comment]
Tak, metody Allegro One Punkt oraz Allegro One Box mają obecnie dwóch przewoźników - UPS oraz DPD. W związku z tym do nazw dodane zostały stosowne oznaczenia. Pobierając konkretne zamówienie otrzymujesz jednoznaczny identyfikator typu przesyłki (pole "delivery.method.id"), które następnie musisz wskazać podczas [tworzenia przesyłki](https://developer.allegro.pl/documentation/#operation/createNewShipment).
Obecnie dostępne są:
- Allegro One Box, UPS - "0aafb43c-e66a-46ec-9cc4-29bb39ebb483"
- Allegro One Box, DPD - "0b257488-c85d-4507-b967-9b45ffbfa2e8"
- Allegro One Punkt, UPS - "08c15d0b-9b24-4b3f-89a8-7a2b502abe55"
- Allegro One Box, DPD - "09ef5a54-c34d-41ef-908e-9b45ffbfa2e8"
